src/gui/entry/EntryModel.h

```diff
@@ -33,7 +33,8 @@ class EntryModel : public QAbstractTableModel
         ParentGroup = 0,
         Title = 1,
         Username = 2,
-        Url = 3
+        Url = 3,
+        LastModified = 4
```

Comment:
Thoughts on ending LastModified with a trailing comma so that additional columns need not modify the previous last column as you have had to do here?